setup.py

```diff
@@ -81,6 +81,7 @@ def get_nightly_version():
             ],
         },
         extras_require={
+            "aws_batch": ["boto3"],
```

Comment:
We should probably keep this the same ver as https://github.com/pytorch/torchx/blob/main/dev-requirements.txt#L4 (so `boto3==1.24.59`) or remove the pin in `dev-requirements.txt`